sdk/search/azure-search-documents/azure/search/documents/_index/aio/_search_index_client_async.py

```diff
@@ -41,24 +41,16 @@ class SearchIndexClient(object):
             :caption: Creating the SearchIndexClient with an API key.
     """
 
+    _ODATA_ACCEPT = "application/json;odata.metadata=none"  # type: str
```

Comment:
I notice here metadata=none, and in non-async we have metadata=minimal; this delta may be intentional so feel free to ignore me, but I'd be remiss to not call it out.  (My brain was going the direction of "well if you always use the same ODATA_ACCEPT can you put it in the mixin")

Reply:
These values were stipulated by @brjohnstmsft

Reply:
More specifically (I just double checked): the index client gets `none` (both sync and async) and the service client gets `minimal` (both sync and async)